README.md

```diff
@@ -15,18 +15,15 @@ Flexible and easy to use email for Elixir.
 
 - **Built-in support for popular mail delivery services**. Bamboo ships with
   [adapters for several popular mail delivery services, including Mandrill,
-  Mailgun, and SendGrid][available-adapters]. It's also quite easy to write
-  your own delivery adapter if your platform isn't yet supported.
-- **Deliver emails in the background**. Most of the time you don't want or need
-  to wait for the email to send. Bamboo makes it easy with
+  Mailgun, and SendGrid][available-adapters]. It's also quite easy to write your delivery adapter if your platform isn't yet supported.
+- **Deliver emails in the background**. Most of the time you don't want or need to wait for the email to send. Bamboo makes it easy with
```

Comment:
I don't think the text change here, correct? Could we leave this as it was? I think we prefer the text wrapping for legibility.